core/src/main/java/org/apache/calcite/sql/validate/SqlAbstractConformance.java

```diff
@@ -105,10 +105,6 @@ public abstract class SqlAbstractConformance implements SqlConformance {
     return SqlConformanceEnum.DEFAULT.isOffsetLimitAllowed();
   }
 
-  @Override public boolean isRegexReplaceCaptureGroupDollarIndexed() {
```

Comment:
where did the functionality of this move?

Reply:
There is a specific library function for BigQuery. BigQuery only supports the 3 argument signature. BigQuery was the only engine that uses dollar signs rather than backslash.

https://github.com/Bit-Quill/calcite/blob/calcite-6310-regex-replace/core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java#L642

https://github.com/Bit-Quill/calcite/blob/calcite-6310-regex-replace/core/src/main/java/org/apache/calcite/util/BuiltInMethod.java#L590